CONTRIBUTING.md

```diff
@@ -135,4 +135,10 @@ To make this easier, we do appreciate it if folks enable the "Allow edits from m
 
 ## On ideologically motivated changes
 
-This is a purely technical project. As such, it is not an appropriate arena to advertise your personal politics or religious beliefs. Any changes that appear ideologically motivated will be rejected.
+Serenity's goal is to enable collaboration between as many groups as *reasonably* possible, and we welcome contributions that make the project more accessisble to people.
```

Comment:
- `groups` feels weird here. Are we thinking about different minorities, or different groups of people within the project (i.e. people who mostly work on Kernel, people who mostly work on userspace, etc)?
- `*reasonably*` can be misinterpreted too, because it's a personal viewpoint. Who's reason are we thinking about? IMHO either remove this word or make it more specific

I would rephrase this sentence as `Serenity's goal is to enable collaboration between as many folks as possible. We welcome any contributions making the project more accessible.`

Reply:
'folks' is no different to 'groups'. A folk is a group, but it's just a very strange and archaic way to describe groups of people that suggests some kind of ethnic connection to most people. What about just leaving it as it is (it's fine as it is) or saying 'between as many people as possible'? Is Serenity's goal *actually* to 'enable collaboration between as many people as possible', though? Or is the goal to produce software? The original description seems accurate: it is a purely technical project aiming to produce a technical result.